transmart-data/test_studies/Makefile

```diff
@@ -0,0 +1,15 @@
+SUBDIRS := $(wildcard */.)
```

Comment:
Odd choice to start using make scripts here again. Why would we choose make, and why is this part of transmart-data?

Reply:
Make does not bring much here indeed. `bash` would do as well.
The rest of test studies that are used by automatic tests are part of `transmart-data` currently.
So you could create the test ready db with `make postgres_test`.
Please feel free to create a ticket for separating test data from `transmart-data` where would decide on a new approach.